Pcap++/src/DpdkDevice.cpp

```diff
@@ -792,7 +792,7 @@ uint16_t DpdkDevice::receivePackets(MBufRawPacketVector& rawPacketsArr, uint16_t
 	//the following line trashes the log with many messages. Uncomment only if necessary
 	//PCPP_LOG_DEBUG("Captured %d packets", numOfPktsReceived);
 
-	if (unlikely(numOfPktsReceived <= 0))
+	if (unlikely(!numOfPktsReceived))

```

Comment:
is this check really needed ?

the for loop will do nothing and we will return numOfPktsReceived at the end

Reply:
To be honest I didn't check all the code when replacing but you are right. This line is unnecessary